P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+                start="#cmr-archi21-slrefl-es-j3-1-ts373" end="#cmr-archi21-slrefl-es-j3-1-ts430">ok
+                hm for me this presentation was hm <pause dur="PT1S"/> become too fast because it's
+                always the same in our architecture school euh we have not time and hm <pause
+                  dur="PT1S"/> too quickly sorry [...]</u>
+              <kinesic xml:id="cmr-archi21-slrefl-es-j3-1-a192" who="#romeorez"
+                start="#cmr-archi21-slrefl-es-j3-1-ts376" end="#cmr-archi21-slrefl-es-j3-1-ts377"
+                type="body" subtype="kinesics">
+                <desc>
+                  <code>eat(popcorn)</code>
+                </desc>
+              </kinesic>
+              <!-- more bodily activities of avatars -->
+              <post modality="written" generatedBy="human" xml:id="cmr-archi21-slrefl-es-j3-1-a195"
+                who="#tfrez2" start="#cmr-archi21-slrefl-es-j3-1-ts380"
+                end="#cmr-archi21-slrefl-es-j3-1-ts381" type="chat-message">
+                <p>it went too quickly?</p>
+              </post>
+            </body>
+          </text>
+        </egXML>
+      </p>
+      <p>Note that the spoken utterance <gi>u</gi> represents a speaker turn that was transmitted
+        via an audio channel of the application that is continuously open during a session, whereas
+        a spoken <gi>post</gi> represents a spoken message that has been recorded in private and
+        been posted to the CMC server as a whole. See section <ptr target="#CMCUnits"/>.</p>
+    </div>
+  </div>
+  <div xml:id="CMCmetadata">
+    <head>Documenting CMC (and providing general metadata)</head>
+    <div xml:id="CMCCorpusSource">
+      <head>Documenting the Source of a Corpus of CMC data</head>
+      <p>The <gi>teiHeader</gi> of the corpus should contain metadata about the CMC platform(s),
+        e.g. its name, information about its owner (often a company) including their address or
+        location, the URL of the server where the CMC data were collected from, or the filename of a
+        database dump that was used as a source. Metadata about the project responsible for
+        collecting the data and building the corpus, if applicable, should be recorded as well.</p>
+      <p>The following example shows the <gi>sourceDesc</gi> of a Twitter corpus. <egXML
+          xmlns="http://www.tei-c.org/ns/Examples">
+          <sourceDesc>
+            <biblFull>
+              <titleStmt>
+                <title>Twitter Sample</title>
+              </titleStmt>
+              <publicationStmt>
+                <distributor>Twitter International Company</distributor>
+                <address>
+                  <addrLine>1 Cumberland Place</addrLine>
+                  <addrLine>Fenian Street</addrLine>
+                  <addrLine>Dublin 2</addrLine>
+                  <postCode>D02 AX07</postCode>
+                  <country>Ireland</country>
+                </address>
```

Comment:
Isn't the web address (https://twitter.com/) in such a case more important than the postal address that really nobody knows? 

And perhaps the date is important? Twitter from e.g. 2015 is a different platform than X/Twitter from 2024.

Reply:
I added `<ptr>` and `<date>` in 759c2adba.